docs/index.html

```diff
@@ -10,25 +7,19 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
     <style>
-        body,
-        html {
+        body, {
             background-color: black;
-            background-image: url(Dentonvale_newtext.png);
-            background-size: 100%;
-            /* background-position: center; */
-            background-repeat: no-repeat;
-            background-size: cover;
         }
     </style>
 </head>
 
-<body>
+<body background="https://scontent-ort2-1.xx.fbcdn.net/v/t1.15752-9/82206523_598760807354715_3062622998457483264_n.jpg?_nc_cat=108&amp;_nc_oc=AQmz8B8dgv4wL6NhESGKDm0wC4aNsO7EGU_gaByEVqmZVjrfBPOOo5mwWm2qR7XugDo&amp;_nc_ht=scontent-ort2-1.xx&amp;oh=8bcc6c7de28b3359e2e41a294e732c9d&amp;oe=5E900157">
```

Comment:
it's probably a bad idea to source this image from facebook's cdn. 

Why is this in body element instead of just replacing the image in the already existing css in the <style> tag?

Reply:
also, what is this?